woocommerce/woocommerce
# Remove deprecated i18n/states directory

### Changes proposed in this Pull Request:

This PR removes the deprecated i18n/states directory.

Since PR #22339, more specifically commit f1afbcf03a704f86fed0e5e7ef153ea0be9d900c, the list of states for each country is defined in i18n/states.php and the directory i18n/states/ is not used anymore. It is not clear from the PR and commit descriptions if i18n/states/ was left on purpose or not. I'm assuming it was not on purpose as it doesn't seem to be used anywhere.

cc @mikejolley @claudiosanches 

### How to test the changes in this Pull Request:

1. Test that WooCommerce is still loading states for a giving country correctly. For example, go to the checkout page and select a country and make sure its list of states is loaded.
2. Check the code to verify that the files inside `i18n/states/` are not used anymore.

### Changelog entry

> Remove deprecated i18n/states directory
